Dark.json

```diff
@@ -54719,7 +54728,7 @@
         ],
         "lightSideIcons": 1,
         "darkSideIcons": 2,
-        "gametext": "Dark:  Once per game, if you occupy three battlegrounds, may retrieve a Black Sun agent into hand..  Light:  Once per game, if you control, may retrieve [Reflections II] Dash into hand."
```

Comment:
Is this an errata to this card?  Looking in scomp link, the card image does not match the new text.

Reply:
Re-downloaded the Set 9 PDF and looked at the current state.  You're right that this game text is wrong.

I will submit an updated PR tonight.

Reply:
Oh wait.  The red line is the old line.  Per the Set 9 PDF, text was errata'd.
![image](https://user-images.githubusercontent.com/5210043/108268054-07edd100-713a-11eb-87ca-766a9c907871.png)

Reply:
It seems there may be other issues then, I just checked card at random.

Reply:
Okay, so the scomp link images are out of date then.  Got it.